modules/gitops-release-notes-1-10-0.adoc

```diff
@@ -0,0 +1,118 @@
++
+[NOTE]
+====
+You need to installed the {gitops-title} Operator in the default `openshift-gitops-operator` namespace, in order to have these metrics automatically available.
+====
+
+* With this update, the {gitops-title} Operator has been enhanced to feature human-readable timestamps in its logs. Previously, timestamps were being presented in a Unix epoch format, but they have now been changed to RFC3339 format, for example: 2023-06-27T07:12:48-04:00, to improve overall readability. link:https://issues.redhat.com/browse/GITOPS-2898[GITOPS-2898]
+
+* With this update, the default Argo CD instance in the `openshift-gitops` namespace will have restricted permissions for non-admin users by default. This will improve security since non-admin users will no longer have access to sensitive information. However, as an administrator, you can set permissions and grant non-admin users access to the resources managed by the default `openshift-gitops` Argo CD instance by configuring your Argo CD RBAC. This change only applies to the default {gitops-title} instance. link:https://issues.redhat.com/browse/GITOPS-3032[GITOPS-3032]
+
+* With this update, the default installation namespace for {gitops-title} Operator has been changed to its own namespace called `openshift-gitops-operators`. You can still choose the old default installation namespace, `openshift-operators`, through a drop down menu available in the OperatorHub UI at installation time. You can also enable cluster monitoring on this namespace by selecting the check box, which will make the operator's performance metrics accessible within the {product-title} web console. link:https://issues.redhat.com/browse/GITOPS-3073[GITOPS-3073]
++
+[NOTE]
+====
+The {gitops-title} Operator's metrics will only be available when the operator is installed in the namespace other than `openshift-operators`.
+====
+
+* Before this update, there was no option for scaling the application controller replicas dynamically. Now, you can dynamically scale the number of application controllers based on the number of clusters managed by each application controller. link:https://issues.redhat.com/browse/GITOPS-3287[GITOPS-3287]
+
+* Before this update, there was no option for choosing an algorthim for distributing the destination clusters equally across the different application controller shards. Now, you can set the sharding algorithm to round-robin, which would distribute clusters equally across the different application controller shards so that the synchronization load is spread equally among the shards. link:https://issues.redhat.com/browse/GITOPS-3288[GITOPS-3288]
+
+* With this update, the {gitops-title} Operator will exports a few custom metrics that will allow you to track the performance of the operator. These metrics include:
```

Comment:
@masauso-lungu Refer to https://redhat-documentation.github.io/supplementary-style-guide/#lead-in-sentences

As per SSG and IBM:

- Use a complete sentence for the lead-in sentence to reduce ambiguity and support translation.
- Lead-in wording: Generally, use a sentence to introduce a list that is not a procedure. Translation consideration: To avoid possible translation problems, use complete sentences and not phrases to introduce lists. Machine translators often translate at the sentence-level and return the best results when they have a
complete sentence to parse.